README.md

```diff
@@ -6,18 +6,18 @@ Redis::ScriptCache - Cached Lua scripts on a Redis server
 
     use Redis;
     use Redis::ScriptCache;
-    
+
```

Comment:
Is this removal of trailing whitespace by any chance accidentally breaking the synopsis into multiple code sections? Or does the Dist::Zilla plugin get that right?

Edit: Of course, I meant the corresponding change in the source that generates README.md.